### PR TITLE
Update DataGrid Getting-started.md file as per the documentation format.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Updated the SfDataGrid Getting-started.md file as per the documentation format.

### CODE REVIEW

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

File: MAUI/DataGrid/Getting-started.md [hunk 1]
```diff
@@ -8,48 +8,33 @@ documentation: ug
...
- {% tabs %}
-{% highlight xaml tabtitle="MainPage.xaml" %}
+Before proceeding, ensure the following are in place:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
 
-<ContentPage   
-    . . .
-    xmlns:syncfusion="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid">
+## Step 1: Create a New MAUI Project
```
Comment (PaulAndersonS) [inline MAUI/DataGrid/Getting-started.md:25]: Step 1: Create a New MAUI Project

Visual Studio

Go to File > New > Project and choose the .NET MAUI App template.
Name the project and choose a location, then click Next.
Select the .NET framework version and click Create.
Visual Studio Code

Open the command palette by pressing `Ctrl+Shift+P`` and type .NET:New Project and enter.
Choose the .NET MAUI App template.
Select the project location, type the project name and press enter.
Then choose Create project

Comment (ashok-kuvaraja) [reply MAUI/DataGrid/Getting-started.md:25]: Modified

**Review by jayaleshwari — CHANGES_REQUESTED** [review]
Please address review comments

File: MAUI/DataGrid/Getting-started.md [hunk 2]
```diff
@@ -83,14 +77,49 @@ namespace GettingStarted
...
+{% endtabs %}
+
+## Step 5: Define the View Model
 
-## Create DataModel for the SfDataGrid
+### Data Model
 
 The SfDataGrid is a data-bound control. Hence, a data model should be created to bind it to the control. 
 
 Create a simple data source as shown in the following code example in a new class file, and save it as OrderInfo.cs file:
```
[the remark this answers is not in the document]
Comment (ashok-kuvaraja) [reply MAUI/DataGrid/Getting-started.md]: Highlighted the class name

File: MAUI/DataGrid/Getting-started.md [hunk 3]
```diff
@@ -144,6 +173,8 @@ public class OrderInfo
 
 N> If you want your data model to respond to property changes, implement the `INotifyPropertyChanged` interface in your model class.
 
+### View Model
+
 Create a model repository class with OrderInfo collection property initialized with the required number of data objects in a new class file as shown in the following code example and save it as OrderInfoRepository.cs file:
```
[the remark this answers is not in the document]
Comment (ashok-kuvaraja) [reply MAUI/DataGrid/Getting-started.md]: Highlighted the class name

**Review by jayaleshwari — APPROVED** [review]
Changes are fine.

**Pull request merged**